test/uiSchema_test.js

```diff
@@ -195,6 +195,41 @@ describe("uiSchema", () => {
       });
     });
 
+    describe("color", () => {
+      const uiSchema = {
+        foo: {
+          "ui:widget": "color"
+        }
+      };
+
+      it("should accept a uiSchema object", () => {
+        const {node} = createFormComponent({schema, uiSchema});
+
+        expect(node.querySelectorAll("[type=color]"))
+          .to.have.length.of(1);
+      });
+
+      it("should support formData", () => {
+        const {node} = createFormComponent({schema, uiSchema, formData: {
+          foo: "a"
+        }});
+
+        expect(node.querySelector("[type=color]").value)
+          .eql("a");
+      });
+
+      it("should update state when text is updated is checked", () => {
+        const {comp, node} = createFormComponent({schema, uiSchema, formData: {
+          foo: "a"
+        }});
+
+        return SimulateAsync().change(node.querySelector("[type=color]"), {
+          target: {value: "b"}
+        })
+          .then(() => expect(comp.state.formData).eql({foo: "b"}));
+      });
+    });
```

Comment:
Oh, it's an uiSchema test, I see. Well so we need to add some test in StringField_test to check for the field behavior, like validation (look at what we did for date widgets for instance).